Dockerfile

```diff
@@ -14,8 +14,8 @@ ARG GIT_MERGE_BRANCH
 WORKDIR /build/
 ADD . /build/
 RUN echo "Starting the script sections" && \
-		pub get --packages-dir && \
-		xvfb-run -s '-screen 0 1024x768x24' pub run dart_dev test --pub-serve --web-compiler=dartdevc -p chrome -p vm && \
-		echo "Script sections completed"
+	pub get --packages-dir && \
+	xvfb-run -s '-screen 0 1024x768x24' tool/test.sh && \
```

Comment:
Why run tests in workiva build as well as travis CI?

Reply:
I'm not sure, maybe RM asked us to?

Reply:
AFAIK, all RM cares about is that a `pub get` is run so that the dependency check can take place.